racingcar-java/src/main/java/racingcar/Race.java

```diff
@@ -0,0 +1,27 @@
+package racingcar;
+
+import racingcar.Car.Cars;
+import racingcar.view.OutputView;
+
+public class Race {
+    private final int progressTotal;
+    private Cars cars;
```

Comment:
```suggestion
    private final Cars cars;
```
위와 마찬가지로 Cars참조값은 변경될 여지를 주면 되지 않기 때문에 final을 붙이는 것이 더 좋아보입니다.